independent-publisher-connectors/Informa/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1976 @@
+            "x-ms-visibility": "important"
+          },
+          {
+            "name": "product",
+            "default": "INFORME_CONECTOR",
+            "in": "query",
+            "type": "string",
+            "required": true,
+            "x-ms-summary": "Product",
+            "description": "Product ID",
+            "x-ms-visibility": "advanced"
+          },
+          {
+            "name": "language",
+            "default": "es",
+            "in": "query",
+            "type": "string",
+            "required": false,
+            "x-ms-summary": "Language",
+            "description": "Language in which to receive the decoding of literals.",
+            "x-ms-visibility": "advanced"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "campoCodificadoRespuesta": {
+                  "type": "object",
+                  "properties": {
+                    "valor": {
+                      "type": "integer",
+                      "format": "int32",
+                      "description": "code value",
+                      "title": "code value"
+                    },
+                    "tablaDecodificacion": {
+                      "type": "string",
+                      "description": "Decoding table",
+                      "title": "Decoding table"
+                    },
+                    "literal": {
+                      "type": "string",
+                      "description": "literal"
+                    }
+                  },
+                  "description": "campoCodificadoRespuesta"
+                },
+                "datosPeticion": {
+                  "type": "object",
+                  "properties": {
+                    "productoSolicitado": {
+                      "type": "string",
+                      "description": "Requested product",
+                      "title": "Requested product"
+                    },
+                    "numeroVersionTablasInformacionComercial": {
+                      "type": "number",
+                      "format": "float",
+                      "description": "Comercial information table version number",
+                      "title": "Comercial information table version number"
+                    },
+                    "parametrosCliente": {
+                      "type": "object",
+                      "properties": {
+                        "idioma": {
+                          "type": "string",
+                          "description": "Language",
+                          "title": "Language"
+                        },
+                        "username": {
+                          "type": "string",
+                          "description": "Username",
+                          "title": "Username"
+                        },
+                        "formato": {
```

Comment:
Kindly correct the spelling "formato"

Reply:
Kindly post your connector in English, this will be translated to other languages later.

Reply:
The API requires specific parameters, as indicated. This is a Spanish solution that provides public information about Spanish companies.

Reply:
As per the public documentation link mentioned below, your connector, titles, descriptions, operations, summaries everything should be in English. 

https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions